database/src/main/resources/db/migration/V59__ektefelle_alder.sql

```diff
@@ -0,0 +1,11 @@
+update behandling set behandlingsinformasjon = behandlingsinformasjon #- '{bosituasjon, epsFnr}';
+update behandling set behandlingsinformasjon = jsonb_set(behandlingsinformasjon, '{bosituasjon, epsAlder}', 'null');
```

Comment:
Hva representerer epsAlder her? Antall år eller fødselsdato? Det holder kanskje med antall år på søknadsdato? Selv om det har endret seg ved behandlingstidspunktet? Eller endrer seg ila. stønadsperioden?

Reply:
Den representerer alder i år, ja. Ideelt sett så burde jo ikke bosituasjon dobbeltlagre noe info om EPS i det hele tatt, men heller bare hente det fra satt EPS, men det krever en liten ekstra opprydning i Behandlingsinformasjon.

Men godt poeng om hva vi evt. gjør dersom man går fra 66 til 67 underveis i behandlingen, det har vi nok ikke noen god rutine for. Vi har vel forsåvidt også et lignende problem i dag med Fnr, men hvor det saksbehandler ser rundt sats potensielt kan endre seg til brevet går ut osv.. skal ta en liten sjekk med Tor Erik om dette er viktig eller ei.